cmd/search_log.go

```diff
@@ -137,3 +179,129 @@ func clicksFn(cmd *cobra.Command, args []string) {
 	}
 	printCsv(records)
 }
+
+func latencyFn(cmd *cobra.Command, args []string) {
+	logger := initLogger()
+	headers := []string{
+		"#", "SearchId", "Term",
+	}
```

Comment:
Make this const in this file, then below use len of this slice.

Reply:
array constant?